ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -9,6 +9,37 @@
     ViewData["RoutingPuzzleId"] = ViewContext.RouteData.Values["puzzleId"];
 }
 
+@{
+    var mailtoUrl = "mailto:";
+    
+    // to: Add the proper support agent as recipient
+    mailtoUrl += Model.PuzzleState?.Puzzle.SupportEmailAlias ?? (Model.Event?.ContactEmail ?? "puzzhunt@microsoft.com");
```

Comment:
This is probably a good thing, but we'll need to go through the SupportEmailAlias fields we currently have set so they go to the right place

Reply:
Yup :) I highlighted this in the tech meeting and the room said this logic is the right one (and we just need to be good about checking that stuff)